CONTRIBUTING.md

```diff
@@ -182,5 +182,12 @@ When squashing-and-merging:
 -   Do a final review of the one-line PR summary, ensuring that it meets the
     guidelines (e.g., "blob: add more blobbing") and accurately describes the
     change.
+-   If the PR includes a breaking change, which should be declared via a commit
+    with `BREAKING_CHANGE_OK` in it (Travis checks for this), add
+    `BREAKING_CHANGE` to the PR summary (e.g., "blob: BREAKING_CHANGE remove old
```

Comment:
Can these markers be put in the body of the commit/PR instead of the subject/title? It makes it really easy to blow out the 72 character suggestion for the first line. One could also imagine using [Git trailers](https://git-scm.com/docs/git-interpret-trailers) for this sort of metadata so that you could programmatically discover them.

Reply:
I want them to be easily visible when reviewing commits for a release, e.g.:
https://github.com/google/go-cloud/compare/v0.11.0...v0.12.0

Is there a way to view that list with full commit bodies?

I haven't heard of commit trailers, but are they supported via the Github UI for squash-and-merge?

Reply:
They are supported from GitHub UI in the sense that it is purely a commit message convention and you could enter them in the commit message when squashing.

One could write a script pretty easily that would find commits with a trailer in a given commit range. If not being able to do scan in the UI is a dealbreaker, then I'm fine with keeping the requirements the way they are. Figured I'd mention other tricks in the arsenal in case they were helpful in solving the problem.

Reply:
Let's see how it goes with this and we can adjust if needed.